### DISCUSSION TITLE
种子下载完成后无法正常做种

### DISCUSSION THREAD

**Discussion by ZzzzRyan** [filecxx/FileCentipede#716] [Q&A] [answered]
我通过磁力链接下载完成后的文件无法正常做种，具体表现为：

1. 没有上传流量；
2. “做种中”状态只会持续几秒钟，而后文件就会被删除。

演示 GIF ：

![动画](https://github.com/filecxx/FileCentipede/assets/107601663/c6c44c00-89e2-4e9a-8157-b1e097c84dd4)

我的设置：

![image](https://github.com/filecxx/FileCentipede/assets/107601663/c3a359f0-79c5-4e5d-a83f-74e3e7fcc411)

请问是否是我哪里没有设置正确或者操作失当导致的？

**Comment by filecxx** [comment, marked as answer]
![image](https://github.com/filecxx/FileCentipede/assets/100348948/3e93cbaf-ed89-40b3-a845-5f3ebacfb570)

 ![image](https://github.com/filecxx/FileCentipede/assets/100348948/8a78422c-dc5b-4c4e-a919-7bad972fb179)

这个组合应该有BUG. 另外需要检查下任务是否有单独的设置
![image](https://github.com/filecxx/FileCentipede/assets/100348948/b7becebd-83ff-4f4f-940f-e3f3e89c2741)
![image](https://github.com/filecxx/FileCentipede/assets/100348948/ea09e96a-bc12-41ca-9d60-aeaa0c11fbaf)



如果没有其他人在下载这个文件或者在(二级路由 || 路由没有公网IP) 的情况下没有上传流量是正常状态.

**Reply by ZzzzRyan** [reply]
感谢！确实，换了个组合就可以正常做种了。
